3rdParty/rocksdb/6.2/util/channel.h

```diff
@@ -60,7 +60,7 @@ class channel {
 
  private:
   std::condition_variable cv_;
-  std::mutex lock_;
+  mutable std::mutex lock_;
```

Comment:
This is RocksDB source code. If we fix it here, it will come back once we upgrade RocksDB.